src/pages/Admin/Users/index.jsx

```diff
@@ -240,7 +240,7 @@ export default class Users extends Component {
         onCancel={userStore.hideModal}
         hideFooter
       >
-        <form className="formContent" onSubmit={userStore.createOrModify} method="post">
+        <form className="formContent" onSubmit={e => userStore.createOrModify(e)} method="post">
```

Comment:
no need to change

Reply:
This change for prevent page refresh。
 createOrModify = async e => {
    e.preventDefault();
    ...
}